loops-arrays-objects.js

```diff
@@ -131,5 +161,17 @@ const students = [
 // ]
 // ==========================================
 
-
-
+const neighborhoodMap = {
+    '3513': "Pijlsweerd",
+    '3514': "Vogelenbuurt",
+    '3512': "Binnenstad",
+    '3531': "Lombok",
+    '3572': "Wittevrouwen",
+    '3581': "Oudwijk",
+    '3583': "Schildersbuurt",
+};
+
+students.forEach(student =>{
```

Comment:
De forEach-methode is hier goed toegepast en zorgt voor een duidelijke en eenvoudige implementatie.